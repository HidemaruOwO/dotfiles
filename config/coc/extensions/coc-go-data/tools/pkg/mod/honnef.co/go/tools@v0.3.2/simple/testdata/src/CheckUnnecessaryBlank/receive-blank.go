package pkg

func fn2() {
	var ch chan int
	<-ch
	_ = <-ch //@ diag(`unnecessary assignment to the blank identifier`)
	select {
	case <-ch:
	case _ = <-ch: //@ diag(`unnecessary assignment to the blank identifier`)
	}
	x := <-ch
	y, _ := <-ch, <-ch
	_, z := <-ch, <-ch
	_, _, _ = x, y, z
}
