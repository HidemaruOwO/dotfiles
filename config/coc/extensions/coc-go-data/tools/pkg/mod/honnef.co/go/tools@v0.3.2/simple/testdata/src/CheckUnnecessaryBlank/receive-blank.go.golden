package pkg

func fn2() {
	var ch chan int
	<-ch
	<-ch //@ diag(`unnecessary assignment to the blank identifier`)
	select {
	case <-ch:
	case <-ch: //@ diag(`unnecessary assignment to the blank identifier`)
	}
	x := <-ch
	y, _ := <-ch, <-ch
	_, z := <-ch, <-ch
	_, _, _ = x, y, z
}
